web/src/api.util.js

```diff
@@ -6,9 +6,14 @@
  * @returns {string} url for  the api
  */
 export const apiUrl = (endpoint) => {
-  const protocol = process.env.REACT_APP_PROTOCOL || "http";
-  const domain = process.env.REACT_APP_API_DOMAIN || "api.lvh.me";
-  return `${protocol}://${domain}/${endpoint}`;
+  const baseUrl =
+    process.env.NODE_ENV === "production"
+      ? process.env.REACT_APP_API_DOMAIN
```

Comment:
This env variable doesn't include the protocol  or the leading forward slashes. This may cause issues during the deployment.

Reply:
Not sure if this is related, but I'm still getting the "failed to save" error on the deployment preview linked upthread. 
![Uploading Screen Shot 2022-02-01 at 11.13.31 AM.png…]()

Reply:
Interesting, this appears to be a CORS error